src/docs.js

```diff
@@ -135,8 +135,60 @@ async function documenter(options) {
   let output = {
     tgz,
   };
+  return output;
+}
+
+async function downloader(options) {
+  options = _.defaults({}, options, {
+    credentials: {},
+    project: null,
+  });
+
+  let auth = new client.Auth({
+    credentials: options.credentials,
+  });
+
+  let creds = await auth.awsS3Credentials('read-write', 'taskcluster-raw-docs', options.project + '/');
```

Comment:
I suppose these only need to be read-only, but it's probably not super important to get that fixed at this very moment. We'd create a second set of creds for this that are read-only and pass them in for this part.